network/state.go

```diff
@@ -17,28 +17,10 @@
 package network
 
 import (
-	"github.com/ChainSafe/gossamer/common"
 	"github.com/ChainSafe/gossamer/core/types"
 )
 
 // BlockState interface for block state methods
 type BlockState interface {
 	LatestHeader() *types.Header
-	AddBlock(*types.Block) error
-	SetBlock(*types.Block) error
-}
-
-// NetworkState interface for network state methods
-type NetworkState interface {
-	GetHealth() (*common.Health, error)
-	SetHealth(*common.Health) error
-	GetNetworkState() (*common.NetworkState, error)
-	SetNetworkState(*common.NetworkState) error
-	GetPeers() (*[]common.PeerInfo, error)
-	SetPeers(*[]common.PeerInfo) error
-}
```

Comment:
should we keep `NetworkState` as an empty struct since it will be needed soon for requested block ids?

Reply:
I'm not sure why the network package would need network state- the network package already knowns about its own state.

Reply:
the network service will need network state to set network state

Reply:
okay, either way we should add it in when we need it

Reply:
i can add it back in but i really don't like having unused code.... if a PR needs it, it should add it in. also even if it's deleted it's still always there because git